process-services/2.3/using/process/form.md

```diff
@@ -22,7 +22,7 @@ The form controls for each field determine how the field is displayed and handle
 |Date/Time|The behavior is similar to that of the Date control, with the added capability of allowing selection of a time value.|
 |Dropdown|Allows you to select an item from a displayed list of items.|
 |Typeahead|On entering data, displays filtered information in a list and allows selection of a value.|
-|Amount|Allows you to input data representing an amount of money and to define a currency type.|
+|Amount|Allows you to input data representing an amount of money and to define a currency type.<br><br>The default values are -99999999999999.98 (Min value) to 99999999999999.98 (Max value).<br><br>Decimal places will have max validation at 14 with 2 decimal places. <br><br> Both Min & Max values are subjected to change dynamically based on decimal places enabled and it's value. In absence of user defined values, system defaults will be used. |
```

Comment:
First two need a bit alteration.

The default values with **decimal enabled** are -99999999999999.98 (Min value) to 99999999999999.98 (Max value).
Decimal places will have **max(14) & min(1) validation**.